doc/source/tokudb/tokudb_variables.rst

```diff
@@ -0,0 +1,432 @@
+.. _tokudb_variables:
+
+================
+TokuDB Variables
```

Comment:
Lines 374-432? for tokudb_enable_partial_eviction and tokudb_compress_buffers_before_eviction